azure-functions/src/bindings/durable_activity_context.rs

```diff
@@ -33,6 +33,7 @@ mod tests {
     use crate::rpc::typed_data::Data;
 
     #[test]
+    #[ignore]
```

Comment:
Wrong file.  Should be the `durable_orchestration_context.rs`.

Reply:
This test is fine as we don't parse any data yet.

Reply:
Oops